clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/tags/EntityTagger.groovy

```diff
@@ -42,6 +45,16 @@ interface EntityTagger {
               String value,
               Long timestamp)
 
+  void tag(String cloudProvider,
+           String accountId,
+           String region,
+           String namespace,
+           String entityType,
+           String entityId,
+           String tagName,
+           Object value,
+           Long timestamp)
```

Comment:
I could see `timestamp` being dropped at some point in the future, but happy to see it left for the time being.